### ISSUE TITLE
Bot not Start - Timezone to long

### ISSUE THREAD

**Issue by Askeshine** [Newcomer1989/TSN-Ranksystem#456]
Only Show this -> http://tinyurl.com/yb74k3ll

**Comment by Shad86** [comment]
Windows or Linux?

**Comment by Askeshine** [comment]
Linux.

**Comment by Shad86** [comment]
You can try to debug the Ranksystem to get more errors / hints.

1) Go to file jobs\bot.php and comment the line `error_reporting(0);` out (change to `//error_reporting(0);`)
2) Open a command line and run the jobs\bot.php manual
`php jobs\bot.php`
3) If you get an output, try to solve the issues that are sowing up, or post the ouput here

**Comment by Askeshine** [comment]
[root@centos67 jobs]# php bot.php
PHP Notice:  date_default_timezone_set(): Timezone ID 'America/Argentina/Buenos_                                                     Aire' is invalid in /var/www/html/ranking/other/config.php on line 148
PHP Warning:  DateTime::createFromFormat(): It is not safe to rely on the system                                                     's timezone settings. You are *required* to use the date.timezone setting or the                                                      date_default_timezone_set() function. In case you used any of those methods and                                                      you are still getting this warning, you most likely misspelled the timezone ide                                                     ntifier. We selected the timezone 'UTC' for now, but please set date.timezone to                                                      select your timezone. in /var/www/html/ranking/jobs/bot.php on line 40
PHP Fatal error:  Uncaught exception 'Exception' with message 'DateTimeZone::__c                                                     onstruct(): Unknown or bad timezone (America/Argentina/Buenos_Aire)' in /var/www                                                     /html/ranking/jobs/bot.php:40
Stack trace:
#0 /var/www/html/ranking/jobs/bot.php(40): DateTimeZone->__construct('America/Ar                                                     genti...')
#1 /var/www/html/ranking/jobs/bot.php(81): enter_logfile('/var/www/html/r...', '                                                     America/Argenti...', 5, 'Initialize Bot....')
#2 {main}
  thrown in /var/www/html/ranking/jobs/bot.php on line 40


Config in WebInterface -> http://tinyurl.com/yb2mkmnj

There its "America/Argentina/Buenos_Aire" But Real is "America/Argentina/Buenos_Aires" , only "s" in Buenos_Aire

**Comment by Shad86** [comment]
We found the bug, this will be fixed in the next version

**Comment by Askeshine** [comment]
2018-03-21 06:39:54.331183   CRITICAL  Please check the files pemissions. Shutting down!
2018-03-21 06:39:54.331045   WARNING       File is not writeable /var/www/html/ranking/avatars/occnmpdjdjhcbajkinoljhnmhfhbgfggpofkokeg.png
2018-03-21 06:39:54.330908   WARNING       File is not writeable /var/www/html/ranking/avatars/fihiehegnjbgommiihchfiejlfppakhhfkfoebal.png
2018-03-21 06:39:54.330775   WARNING       File is not writeable /var/www/html/ranking/avatars/hlifcioagdclodbfeckgocbbpoiaamnahfopelkh.png
2018-03-21 06:39:54.330625   WARNING       File is not writeable /var/www/html/ranking/avatars/dnfokigedegmmkjefkknlgffpeglcgkeckchcncf.png
2018-03-21 06:39:54.330494   WARNING       File is not writeable /var/www/html/ranking/avatars/lpcoibnokdcciikehnajgmjnkpacfilcpaijdckd.png
2018-03-21 06:39:54.330364   WARNING       File is not writeable /var/www/html/ranking/avatars/fnjhfjemjpnoomdocojfiodpgefaiedmlbhljdbg.png
2018-03-21 06:39:54.330234   WARNING       File is not writeable /var/www/html/ranking/avatars/icennmkbkjngdoeiaehhomkeoniapdkcmlepaoph.png
2018-03-21 06:39:54.330103   WARNING       File is not writeable /var/www/html/ranking/avatars/ngjfnjebmgjceacjamiminncpcchmnecbgaaodma.png
2018-03-21 06:39:54.329967   WARNING       File is not writeable /var/www/html/ranking/avatars/omegejpfkeiomcmfngkkafenagpmjmkkkfpfokea.png
2018-03-21 06:39:54.329834   WARNING       File is not writeable /var/www/html/ranking/avatars/iaglhkbeankndgmdfpbdlcfcjbaomgomhcikeeca.png
2018-03-21 06:39:54.329668   WARNING       File is not writeable /var/www/html/ranking/avatars/jkafahfepebnajaebdbppkjijelajhfplhaccmdh.png
2018-03-21 06:39:54.329430   WARNING       File is not writeable /var/www/html/ranking/avatars/jjpagapmhenbbddcfhlgkjjhnpnopfimcfgemcld.png
2018-03-21 06:39:54.329256   WARNING       File is not writeable /var/www/html/ranking/avatars/moclkjlbokgpgfpkbabaefoiilbaidkiedekfbdf.png
2018-03-21 06:39:54.309773   WARNING       File is not writeable /var/www/html/ranking/logs/pid

**Comment by bykidi** [comment]
>chmod -R 777 /var/www/html/rankings/

**Comment by Shad86** [comment]
> chmod -R 777 /var/www/html/rankings/

Is not really a good advise

only do:

`chmod 777 tsicons/ avatars/ logs/ update/ other/dbconfig. php -Rf`

**Comment by bykidi** [comment]
no idea why but
if i don't use 777 on files + 755 on folders - ranksystem is not working at all.
it even blame my robots.txt which is not a part of ranksystem

**Comment by Shad86** [comment]
looks for me, like a misconfigured webserver, because why should the robot.txt effect the bot, doesn't make sense at all...

**Comment by JVMerkle** [comment]
This can happen when the wrong user (like root) is owning the files.
Provided that your webserver-user is `www-data`:
```bash
chown www-data:www-data -R ts3rankbot/
chmod 754 -R ts3rankbot/
```
You also should't be doing:
```bash
[root@machine]$ php bot.php
```
Use a shell with your `www-data` user instead:
```bash
[root@machine]$ su -s /bin/bash - www-data
[www-data@machine]$ some-command
```

**Comment by Shad86** [comment]
deleted some comments, that are non related - if you have still an issue open a new issue

**Comment by Newcomer1989** [comment]
For information.. The correction is not part of release 1.2.6.

It will be fixed with release 1.2.7

**Comment by Newcomer1989** [comment]
solved with release version 1.2.7
